CHANGELOG.md

```
@@ -14,6 +14,10 @@ This changelog summarizes releases for the following packages:
 
 ## @effectify/prisma
 
+## 1.1.15 (2026-09-02)
+
+## @effectify/prisma
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)